SU2_CFD/src/integration/CIntegration.cpp

```diff
@@ -39,53 +40,54 @@ CIntegration::CIntegration(CConfig *config) {
   Convergence = false;
   Convergence_FSI = false;
   Convergence_FullMG = false;
-  Cauchy_Serie = new su2double [config->GetCauchy_Elems()+1];
+  Cauchy_Serie.resize(config->GetCauchy_Elems()+1, 0.0);
   InitResidual = 0.0;
 }
 
-CIntegration::~CIntegration(void) {
-  delete [] Cauchy_Serie;
-}
-
 void CIntegration::Space_Integration(CGeometry *geometry,
                                      CSolver **solver_container,
                                      CNumerics **numerics,
                                      CConfig *config, unsigned short iMesh,
                                      unsigned short iRKStep,
                                      unsigned short RunTime_EqSystem) {
   unsigned short iMarker, KindBC;
-  
+
   unsigned short MainSolver = config->GetContainerPosition(RunTime_EqSystem);
   bool dual_time = ((config->GetTime_Marching() == DT_STEPPING_1ST) ||
                     (config->GetTime_Marching() == DT_STEPPING_2ND));
 
   /*--- Compute inviscid residuals ---*/
-  
+
   switch (config->GetKind_ConvNumScheme()) {
     case SPACE_CENTERED:
-      solver_container[MainSolver]->Centered_Residual(geometry, solver_container, numerics[CONV_TERM], config, iMesh, iRKStep);
+      solver_container[MainSolver]->Centered_Residual(geometry, solver_container, numerics, config, iMesh, iRKStep);
```

Comment:
I had to remove the `[CONV_TERM]` bit because I need to pass an array of numerics such that each thread picks one... which gives a bit more freedom to those functions than what they should have IMO, but there we are...